src/credentials/sso/ssoAccessTokenProvider.ts

```diff
@@ -4,93 +4,103 @@
  */
 
 import * as vscode from 'vscode'
-import { ClientRegistration } from './clientRegistration'
-import { AccessToken } from './accessToken'
+import { SSOOIDC } from 'aws-sdk'
+import { SsoClientRegistration } from './ssoClientRegistration'
+import { SsoAccessToken } from './ssoAccessToken'
 import { DiskCache } from './diskCache'
-import { Authorization } from './authorization'
 import { getLogger } from '../../shared/logger'
+import { StartDeviceAuthorizationResponse } from 'aws-sdk/clients/ssooidc'
 
 const CLIENT_REGISTRATION_TYPE = 'public'
+const CLIENT_NAME = `aws-toolkit-vscode-${Date.now()}`
```

Comment:
`aws-toolkit-vscode` is the constant component that this module has in common with the disk cache `registrationFilename` method. (This can be shared). 

The datestamp is relevant for the time when credentials are requested. Putting the date into the constant will only issue the timestamp when the extension was activated, and use that for the duration of the VS Code session. Is this acceptable?

Reply:
The timestamp is not required and can be left out.  It is not used elsewhere in the toolkit.